Makefile

```diff
@@ -96,8 +96,11 @@ $(TARFILE): NODE_ENV=production
 $(TARFILE): $(WEBPACK_TEST) cockpit-$(PACKAGE_NAME).spec
 	if type appstream-util >/dev/null 2>&1; then appstream-util validate-relax --nonet *.metainfo.xml; fi
 	mv node_modules node_modules.release
-	touch -r package.json package-lock.json
+	touch -r package.json package-lock.json ${NODE_MODULES_TEST}
 	touch dist/*
+	echo tar czf cockpit-$(PACKAGE_NAME)-$(VERSION).tar.gz --transform 's,^,cockpit-$(PACKAGE_NAME)/,' \
```

Comment:
Is it needed? IIRC `make` prints the executed commands...

And if it is really needed I'd avoid duplicating it, it can easily get out of sync.

```
TAR_COMMAND=tar czf ...
echo $(TAR_COMMAND)
$(TAR_COMMAND)
```
```

Reply:
Sorry, it was a leftover.